app-modules/monitoring/src/Http/Middleware/RecordHttpMetrics.php

```
@@ -0,0 +1,73 @@
+        }
+
+        $start = hrtime(true);
+
+        /** @var Response $response */
+        $response = $next($request);
+
+        $duration = (hrtime(true) - $start) / 1e9;
+
+        try {
+            $registry = app(CollectorRegistry::class);
+            $ns = config('prometheus.default_namespace', 'app');
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $ns. Configured minimum length is 3. (undefined)

(ShortVariable)